adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/AdapterFactory.java

```diff
@@ -19,7 +18,7 @@
 import static java.util.Map.entry;
 
 @Service
-public class AdapterRepository {
+public class AdapterFactory {
```

Comment:
Renamed it because Repository keyworkd is meant for persistence stuff in SpringBoot